serverless/aws/sam/src/main/scala/sttp/tapir/serverless/aws/sam/AwsSamOptions.scala

```diff
@@ -12,3 +14,15 @@ case class AwsSamOptions(
 sealed trait FunctionSource
 case class ImageSource(imageUri: String) extends FunctionSource
 case class CodeSource(runtime: String, codeUri: String, handler: String) extends FunctionSource
+
+case class HttpApiProperties(cors: Option[Cors])
+object HttpApiProperties {
+  case class Cors(
+      allowCredentials: Option[Boolean],
+      allowedHeaders: List[String],
```

Comment:
Hm maybe ... depending how hard it would be. Or maybe simply re-use that class - or is it incompatible? Not sure if a dependency SAM -> interceptor config is correct though ;)